src/udp_wrap.cc

```diff
@@ -233,16 +233,17 @@ void UDPWrap::BufferSize(const FunctionCallbackInfo<Value>& args) {
 
   CHECK(args[0]->IsUint32());
   CHECK(args[1]->IsUint32());
-  int size = static_cast<int>(args[0].As<Uint32>()->Value());
+  uint32_t u_size = static_cast<int>(args[0].As<Uint32>()->Value());
```

Comment:
shouldn't this be `static_cast<uint32_t>` ?

Reply:
actually, `Uint32::Value()` returns a `uint32_t` already so you can remove the cast

Reply:
@targos Thanks! 

I was just focusing on the compiler warning and not giving proper attention to the code I'm afraid. I think it was correct to have the cast to int as this is then checked to make sure that the value can fit into an int before passing it along to `uv_recv_buffer_size` or `uv_send_buffer_size`. Otherwise it will never detect such a situation.